share/p2p/shrexsub/pubsub.go

```diff
@@ -94,9 +94,9 @@ func (s *PubSub) AddValidator(v ValidatorFn) error {
 }
 
 func (v ValidatorFn) validate(ctx context.Context, p peer.ID, msg *pubsub.Message) pubsub.ValidationResult {
-	var pbmsg pb.EDSAvailableMessage
+	var pbmsg pb.RecentEDSNotification
 	if err := pbmsg.Unmarshal(msg.Data); err != nil {
-		log.Debugw("shrex-push: unmarshal error", "err", err)
+		log.Debugw("validator: unmarshal error", "err", err)
```

Comment:
It will be prefixed by logger name "shrex-sub" anyway. I have added "validator" prefix to distinguish this error from another unmarshal error inside subscription